gradle.properties

```diff
@@ -13,3 +13,4 @@ POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
 POM_LICENCE_DIST=repo
 POM_DEVELOPER_ID=mcharmas
 POM_DEVELOPER_NAME=Michal Charmas
+REACTIVE_LOCATION_GMS_API_KEY
```

Comment:
For JitPack to build. It is because of the `build.gradle` file of the `samples` project.